packages/react-components/react-calendar-compat/src/components/CalendarDayGrid/useCalendarDayGridStyles.styles.ts

```diff
@@ -409,5 +455,6 @@ export const useCalendarDayGridStyles_unstable = (props: CalendarDayGridStylePro
         lastTransitionWeekStyles.verticalBackward,
     ),
     dayMarker: mergeClasses(calendarDayGridClassNames.dayMarker, dayMarkerStyles.base),
+    dayTodayMarker: mergeClasses(dayTodayMarkerStyles.base, calendarDayGridClassNames.dayTodayMarker),
```

Comment:
We usually add the classname first and then the other styles. this classname is mainly used for selectors rather than styling.

```suggestion
    dayTodayMarker: mergeClasses(calendarDayGridClassNames.dayTodayMarker, dayTodayMarkerStyles.base),
```